assets/main.scss

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
This overrides the theme's `main.scss`.
I turned the profiles into flexbox.

Reply:
Any specific reason for flexbox? I'm not picky about it, just curious since web dev is not my strong suit.

Reply:
There were a few ways of doing this, but utilizing new browser standard technologies reduces the need for third party javascript. After some quick reading on how to accomplish the desired responsive behavior, flexbox keeps coming up, so I felt it was a good idea to learn the basics of it.